ftdetect/minisnip.vim

```diff
@@ -0,0 +1,13 @@
+" Vim ftdetect file
+" Language:      minisnip
+" Created by:    Robert Hicks <gowther@zoho.com>
+" Last Change:   2017-08-20
+"
+" This simply sets all files that are in the defined
+" directory as 'minisnip'.
+"
+
+" get the snippets directory
+let snippet_dir = expand(g:minisnip_dir) . "*"
+
+execute 'autocmd BufRead,BufNewFile ' . snippet_dir . ' set filetype=minisnip'
```

Comment:
I hadn't planned on changing it any time soon. I'm more focused on writing a deoplete source for it. I develop very casually...heh...